src/APIFunctions/LedSign.js

```diff
@@ -55,7 +53,7 @@ export async function getAllSignLogs() {
 export async function updateSignText(signData) {
   let status = new ApiResponse();
   await axios
-    .post(RPC_API_URL + '/LedSign/updateSignText', { ...signData })
+    .post(LOGGING_API_URL + '/LedSign/sendLEDSign', { ...signData })
```

Comment:
with the renaming of ur api handler, change this back to
```suggestion
    .post(LOGGING_API_URL + '/LedSign/updateSignText', { ...signData })
```